uwblueprint/project-read-frontend
# Build colour-coded status chip component

### Notion ticket link



[Add colour-coding to status column](https://www.notion.so/uwblueprintexecs/Developer-Hub-30e9b6c82af24054b31acf7c5e822c89?p=eccd51f7021d465b9b309e98ff8cd4a1)



### Implementation description

- added a `StatusChip` component that displays a chip with a background colour dependent on the status
- rendered the chip in the "Status" column of the family table component
  - this'll be used in the "current enrolment" and "past enrolment" sections of the sidebar in the future

<img width="1536" alt="Screen Shot 2021-06-28 at 6 37 10 PM" src="https://user-images.githubusercontent.com/37346399/123712227-e5826780-d83f-11eb-8a3b-f4b79355b7ed.png">



### Steps to test

locally:
1. yarn test
2. run the backend on [this PR](https://github.com/uwblueprint/project-read-backend/pull/84), and run `load_initial_data` to get the new statuses
3. open up the registration table or session pages and check out the status column!



### What should reviewers focus on?

code 🧼 🛁 

### Checklist
